Parsers/ArubaClearPass/ArubaClearPass.txt

```diff
@@ -0,0 +1,81 @@
+// Title:           Aruba ClearPass Parser
+// Author:          Microsoft
+// Version:         1.0
+// Last Updated:    12/01/2020
+// Comment:         Initial Release
+//  
+// DESCRIPTION:
+// This parser takes raw Aruba ClearPass logs from a Syslog (CEF) stream and parses the logs into a normalized schema.
+//
+// USAGE:
+// 1. Open Log Analytics/Azure Sentinel Logs blade. Copy the query below and paste into the Logs query window. 
+// 2. Click the Save button above the query. A pane will appear on the right, select "as Function" from the drop down. Enter a Function Name.
+//    It is recommended to name the Function Alias, as Aruba ClearPass
```

Comment:
Please remove space in the name of the function. Function alias is as **ArubaClearPass**

Reply:
Typo corrected